data/json/items/containers.json

```diff
@@ -483,7 +483,7 @@
         "bashing" : 6,
         "to_hit" : -4,
         "contains" : 240,
-        "material" : ["steel"],
+        "material" : "steel",
```

Comment:
Try to keep line counts as low as possible for each PR by avoiding any unrelated or unnecessary changes